### ISSUE TITLE
question about restore_cfg setting

### ISSUE THREAD

**Issue by yushan777** [kijai/ComfyUI-SUPIR#117]
Is it just me or `restore_cfg` does not seem to make much of a difference?  I have used settings of -1, 5, 10, 20, the differences are negligible.

**Comment by kijai** [comment]
I don't think it does anything with the dpmpp samplers, the effect seems pretty small on the EDM samplers too, it's supposed to "restore" some of the original image each step, or something like that, can't say I ever fully understood it.
